templates/project.html

```diff
@@ -149,7 +146,7 @@
     <label class="nav-label" for="toggle">&#9776</label>
     <input type="checkbox" id="toggle"/>
     <div class="menu">
-        <span class="Logo"><a href="#" >EasyGrade</a></span>
+        <span class="Logo"><a href="#" >OpenRubric</a></span>
```

Comment:
Is there a good reason why project.html doesn't use the nav bar from parent_signed_in?

Reply:
That was a mistake, I think I miss that file when I change every document